src/gui/tray/usermodel.cpp

```diff
@@ -915,12 +915,17 @@ bool UserModel::isUserConnected(const int id)
     return _users[id]->isConnected();
 }
 
-QImage UserModel::avatarById(const int id)
+QImage UserModel::avatarById(const int id) const
 {
-    if (id < 0 || id >= _users.size())
+    const auto foundUserByIdIter = std::find_if(std::cbegin(_users), std::cend(_users), [&id](const OCC::User* const user) {
```

Comment:
Is there a reason to use std::cbegin and std::cend over the QList cbegin() and cend() methods?

Reply:
@claucambra It is just a handy wrapper. Is there any downside to using it? It is used a lot in our code-base

Reply:
IMO it makes more sense to use the container's own method for this, but it's a small nitpick :)